pkg/v1/tkg/tkgpackageclient/package_update.go

```diff
@@ -38,6 +38,13 @@ func (p *pkgClient) UpdatePackage(o *tkgpackagedatamodel.PackageOptions, progres
 		}
 	}()
 
+	if o.ValuesFile != "" {
+		if _, err = ioutil.ReadFile(o.ValuesFile); err != nil {
+			err = errors.Wrap(err, fmt.Sprintf("failed to read from data values file '%s'", o.ValuesFile))
+			return
+		}
+	}
+
```

Comment:
This check should have been put after the following block, as we do not want to check twice for the isntall case

```
if pkgInstall == nil {
		if !o.Install {
			err = &tkgpackagedatamodel.PackagePluginNonCriticalError{Reason: tkgpackagedatamodel.ErrPackageNotInstalled}
			return
		}
		progress.ProgressMsg <- fmt.Sprintf("Installing package '%s'", o.PkgInstallName)
		p.InstallPackage(o, progress, tkgpackagedatamodel.OperationTypeUpdate)
		return
	}
```

Reply:
changes updated